docs/source/hooks/common_use_cases.md

```diff
@@ -189,7 +189,7 @@ class AzureSecretsHook:
         }
 ```
 
-Finally, [register the hook](https://kedro.readthedocs.io/en/stable/hooks/introduction.html#registering-your-hook-implementations-with-kedro) in your `settings.py` file:
+Finally, [register the hook](./introduction.md#registering-your-hook-implementations-with-kedro) in your `settings.py` file:
```

Comment:
Oh wow, somehow I'm surprised this works! It's so intuitive and you don't need any of the docutils cross referencing machinery (see https://kedro--2400.org.readthedocs.build/en/2400/hooks/common_use_cases.html#use-hooks-to-load-external-credentials). @stichbury do you happen to have the MyST docs related to this? Or you just did it without thinking much about it? In fact now I'm wondering if this would work in reST as well.

Reply:
I've always done it like this 😆  I guess because I thought it _should_ work and, for once with Sphinx, it did what you'd expect.